dbal/migrate.go

```diff
@@ -70,7 +70,7 @@ func NewPackerMigrationSource(l logrus.FieldLogger, sources []string, loader fun
 
 		var found bool
 		for _, f := range filters {
-			if strings.Contains(source, f) {
+			if filepath.Dir(source) == f {
```

Comment:
this was needed because:

when filter == "migrations/sql/tests", wanted to only include tests under that path, and not tests inside subfolders of "migrations/sql/tests".

without this, I cannot include subfolders, like, e.g. `migrations/sql/tests/cockroach`. `strings.Contains` would match and try to execute the cockroach tests under this scenario on every `migrations/sql/tests` folder (mysql and postgres).